integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -396,6 +396,12 @@ func (v *EthereumKeeperRegistry) Pause() error {
 			return err
 		}
 		return v.client.ProcessTransaction(tx)
+	case ethereum.RegistryVersion_2_1:
+		tx, err = v.registry2_1.Pause(txOpts)
+		if err != nil {
+			return err
```

Comment:
we could probable clean these up by defining an interface that all of the registry versions adhere to (since they all have the same `Pause()` function

Reply:
Good point! I agree the switch-case is growing bigger. I remember we discussed purging tests for older versions in Q4 this year. I would prefer to keep the versions as is for now, as long as we dont grow the versions too quick.